app/views/components/table_rulings.scala.html

```diff
@@ -72,7 +72,7 @@
                                 }
                             </td>
                             <td id="rulings-list-row-@{index}-download">
-                                <a id="rulings-list-row-@{index}-download-link" target="_blank" href="@routes.PdfDownloadController.ruling(c.reference)">@messages("case.ruling.downloadPdf")</a>
+                                <a id="rulings-list-row-@{index}-download-link" target="_self" href="@routes.RulingController.viewRuling(c.reference)">@messages("case.ruling.downloadPdf")</a>
```

Comment:
Is target `_self` not the same as not having the target attribute? It can probably be removed

Reply:
I think it's default dependant on which browser is used, but modern browsers all use it as default so I can safely remove here. I'll get rid as part of tidy up.